modules/ROOT/pages/_partials/commons/common-new-logging-api.adoc

```diff
@@ -0,0 +1,634 @@
+// {param-module} grabs the correct platform name from the related _set_page_context_for_[platform] 
+== Upgrading to the New CBL Logging API
+
+CAUTION: Use of the deprecated and new Logging API at the same time is not supported.
+
+You can find information about the new Couchbase Lite Logging API introduced in Couchbase Lite 3.2.2.
+
+ifndef::is-c[]
+
+For information about the now deprecated earlier version of the Logging API, see xref:3.1@couchbase-lite:{param-module}:troubleshooting-logs.adoc[]
+
+endif::is-c[]
+
+== LogSinks
+
+Couchbase Lite 3.2.2 introduces a new Logging API.
+
+The first thing to note is that the three destinations for logs have been renamed as `LogSinks`, in keeping with common source/sink terminology. 
+
+The `FileLogSink`, the `ConsoleLogSink` and the `CustomLogSink` are all installed in an instance of `LogSinks`. 
+
+All three log sinks are initialized by default. 
+If you want to stop all logging to one of them, though, you can set the sink to null.
+
+For example, you can stop logging to the FileLogSink like this:
+
+`LogSinks.get().setFile(null)`
+
+Couchbase still logs its messages in a handful of named domains and at common log levels: `LogLevel.DEBUG` the most verbose, and `LogLevel.ERROR` only for serious failures.
+
+To get the instance of `LogsSinks`, use the method `LogSinks.get()`.
```

Comment:
`LogSinks.get()` is Java/Android specific and it should be `LogSinks` not `LogsSinks`. 

Maybe, `Accessing LogSinks depends on the platform. In Java, use LogSinks.get() to obtain an instance. In Swift, Objective-C, .NET, and C, LogSinks provides only static methods, without a singleton instance.`

Reply:
Done